docs/example.md

```diff
@@ -68,10 +68,28 @@ The Connector Resolver file (.tdr) is optional. Tableau uses it to create a conn
 The TDR file calls these scripts (described in the following sections): 
 - Connection Builder
 - Connection Properties
-- Connection Matcher
-- Connection Normalizer
 
-The TDR file also includes the driver-resolver section. The <span style="font-family: courier new">driver-resolver</span> is currently only used for ODBC drivers. JDBC connectors can specify the driver name in the URL built by the connection builder JavaScript. 
+The TDR file also contains the connection-normalizer and the driver-resolver section. The <span style="font-family: courier new">connection-normalizer</span> component defines what makes up a unique connection. An example of the component : 
+
+```
+<connection-normalizer>
+    <required-attributes>
+        <setImpersonateAttributes/>
+        <attribute-list>
+            <attr>server</attr>
+            <attr>port</attr>
+            <attr>dbname</attr>
+            <attr>username</attr>
+            <attr>password</attr>
+        </attribute-list>               
+    </required-attributes>
+</connection-normalizer>
+
+```
+`<setImpersonateAttributes/>` adds support for impersonate attributes and should be in every connector.
```

Comment:
`setImpersonateAttributes` can be removed, its done automatically.  I believe samples are already updated.

Reply:
Removed